docs/processors/getting-started.mdx

```diff
@@ -4,99 +4,156 @@ sidebar_label: "Getting Started"
 yourself familiar with [pipeline semantics](/docs/features/pipeline-semantics) is highly recommended.
 
 ![Pipeline](/img/pipeline_example.svg)
 
-Processors are **optional** components in a pipeline, i.e. a pipeline can be started without them. They are always
-attached to a single parent, which can be either a connector or a pipeline. With that, we can say that we have the following
-types of processors:
+Processors are **optional** components in a pipeline (i.e. a pipeline can be started without them), and they are always
+attached to a single parent, which can be either a connector or a pipeline:
 
-1. **Source processors**: these processors only receive messages originating at a specific source connector. Source
+- **Connector processors**:
+  - **Source processors** only receive messages originating at a specific source connector. Source
    processors are created by specifying the corresponding source connector as the parent entity.
-2. **Pipeline processors**: these processors receive all messages that flow through the pipeline, regardless of the
-   source or destination. Pipeline processors are created by specifying the pipeline as the parent entity.
-3. **Destination processors**: these processors receive only messages that are meant to be sent to a specific
+   - **Destination processors** only receive messages that are meant to be sent to a specific
    destination connector. Destination processors are created by specifying the corresponding destination connector as
    the parent entity.
+- **Pipeline processors** receive all messages that flow through the pipeline, regardless of the
+   source or destination. Pipeline processors are created by specifying the pipeline as the parent entity.
+
+
+## Supported processors
+
+When it comes to using a processor either on a connector, or a pipeline, Conduit supports different types:
+
+- [Built-in processors](/docs/processors/builtin) will perform the most common operations you could expect such as filtering fields, replacing fields, posting payloads to a HTTP endpoint, etc. These are already coming as part of Conduit, and you can simply start using them with a bit of configuration. [Check out this document to see everything that's available](/docs/processors/builtin).
+- [Standalone processors](/docs/processors/standalone-processors) are the ones you could write yourself to do anything that's not already covered by the [Built-in](/docs/processors/builtin) ones. [Here's](/docs/processors/standalone-processors) more information about them.
+
+## How to use a processor for your pipeline or connector
+
+In these following examples, we're using the [`json.decode`](/docs/processors/builtin/json.decode), but you could use any other you'd like from our [Built-in](/docs/processors/builtin/) ones, or even [reference](/docs/processors/referencing) your own [Standalone processor]((/docs/processors/standalone-processors)).
```

Comment:
Watch out, you've got an extra set of parentheses in the last link.

Sidenote: since this is a markdown file, you could add some line breaks, then reviewers can target the lines more precisely when posting comments 😉 

```suggestion
In these following examples, we're using the [`json.decode`](/docs/processors/builtin/json.decode), but you could use any other you'd like from our [Built-in](/docs/processors/builtin/) ones, or even [reference](/docs/processors/referencing) your own [Standalone processor](/docs/processors/standalone-processors).
```